sdk/metric/metricdata/metricdatatest/comparisons.go

```diff
@@ -518,14 +523,16 @@ func equalExemplars[N int64 | float64](a, b metricdata.Exemplar[N], cfg config)
 			reasons = append(reasons, notEqualStr("Time", a.Time.UnixNano(), b.Time.UnixNano()))
 		}
 	}
-	if a.Value != b.Value {
-		reasons = append(reasons, notEqualStr("Value", a.Value, b.Value))
-	}
-	if !equalSlices(a.SpanID, b.SpanID) {
-		reasons = append(reasons, notEqualStr("SpanID", a.SpanID, b.SpanID))
-	}
-	if !equalSlices(a.TraceID, b.TraceID) {
-		reasons = append(reasons, notEqualStr("TraceID", a.TraceID, b.TraceID))
+	if !cfg.ignoreValue {
+		if a.Value != b.Value {
+			reasons = append(reasons, notEqualStr("Value", a.Value, b.Value))
+		}
+		if !equalSlices(a.SpanID, b.SpanID) {
+			reasons = append(reasons, notEqualStr("SpanID", a.SpanID, b.SpanID))
+		}
+		if !equalSlices(a.TraceID, b.TraceID) {
+			reasons = append(reasons, notEqualStr("TraceID", a.TraceID, b.TraceID))
+		}
```

Comment:
Should the trace information be ignored here? It doesn't seem like the value.

Reply:
You're right, fixed.